Results.html

```diff
@@ -4,15 +4,16 @@
     <meta charset="UTF-8">
     <title>Cup of Excellence</title>
     <link href='https://fonts.googleapis.com/css?family=Lato:400,300' rel='stylesheet' type='text/css'>
-    <link rel="stylesheet" type="text/css" href="style.css" />
```

Comment:
I think this might be a bit cleaner if we use group many of our CSS files into one import.   That way we can manage all the imports in less locations (separate from HTML).  This way it won't "clutter" up the html (eventually angular).   There are many ways to group them and I leave it to you on how you want to aggregate them (e.g. one file, one for main styles and one for tables, etc).   CHeck out this link....

http://www.dreamincode.net/forums/topic/94486-including-multiple-stylesheets-in-one-css-file/